AccountService.js

```diff
@@ -42,26 +47,42 @@ export class AccountService {
     }
   }
 
+  /**
+   * taken a url and an email creates
+   * an account for a user
+   * @return {Object}
+   * @memberOf AccountService
+   * @param {Object} config
+   * @param {String} config.url
+   * @param {String} config.email
+   */
   async create({url = this.config.urls.base, email}) {
     const response = await axios.post(url, {email}, {headers});
     return response.data;
   }
-
+  /**
+   * takes a url and an id and finds a single account
+   * @memberOf AccountService
+   * @param {Object} config
+   * @param {String} config.baseUrl
+   * @param {String} config.id
+   * @return {Object} an account
+   */
   async get({baseUrl = this.config.urls.base, id}) {
     const response = await axios.get(baseUrl + '/' + id, {headers});
     return response.data;
   }
 
   /**
-   * @method getAll
+   * calls on the index route and returns all
+   * accounts that match the email passed in.
+   * @memberOf AccountService
    * @param {Object} config
    * @param {string} config.baseUrl - the base baseUrl
```

Comment:
We are making these descriptions be proper sentences like in the docs? http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values

That's how we have been doing it already (mostly)

Reply:
- [ ] change to proper capitalization etc.

Reply:
This has not been addressed.  Are we agreed that param descriptions should be like:
```
@param {string} config.baseUrl - The base URL.
```
This is what is shown on the jsdocs docs, and the convention DB has been following elsewhere.